lib/protocol/collection/item_collection_notes_view.ts

```diff
@@ -63,27 +82,25 @@ export class ItemCollectionNotesView {
       direction,
       filter
     );
-    this.tag = tag;
+    this.tagUuid = tag?.uuid;
     this.needsRebuilding = true;
   }
 
   private rebuildList() {
-    const tag = this.tag;
     const notes = this.collection.displayElements(ContentType.Note) as SNNote[];
-    if (tag?.isSmartTag()) {
-      this.displayedList = this.notesMatchingSmartTag(
-        tag as SNSmartTag,
-        notes
-      );
-    } else if (tag) {
-      const taggedNoteUuids = new Set(
-        this.collection.uuidReferencesForUuid(tag.uuid)
-      );
+    if (!this.tagUuid) {
+      this.displayedList = notes;
+      return;
+    }
+
+    const tag = this.collection.find(this.tagUuid) as SNTag;
```

Comment:
I'm not sure that system smart tags (all, trash, archived) will appear inside the collection: https://github.com/standardnotes/snjs/blob/004/lib/services/item_manager.ts#L71

Reply:
You're right, and this broke SNJS. I moved too fast. I pushed a subsequent hotfix afterwards though.